.github/ISSUE_TEMPLATE/03_bug_report.yml

```diff
@@ -0,0 +1,70 @@
+---
+name: 🪲 Bug report
+description: Problems and issues with code in EPAM AI DIAL chat
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      # yamllint disable rule:line-length
+      value: "
+        <img src='https://www.gravatar.com/avatar/2937817a82464ae07b138d8b57a7ccd4?s=40' align='left' width='40' height='40' style='margin-right:10px'>
+        Thank you for finding the time to report the problem!
+
+        We really appreciate the community's efforts to improve EPAM AI DIAL chat.
+        <br clear='left'/>"
+      # yamllint enable rule:line-length
+  - type: input
+    validations:
+      required: true
+    attributes:
+      label: EPAM AI DIAL version
+      # yamllint disable rule:line-length
+      description: >
+        On what version of EPAM AI DIAL are you currently experiencing the issue? You can find the version in the footer of page (e.g. "Version 0.4.2").
+      # yamllint enable rule:line-length
+      placeholder: 0.4.2
+  - type: textarea
+    attributes:
+      label: What happened?
+      description: Describe what happened.
+      placeholder: >
+        Please provide the context in which the problem occurred and explain what happened
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: How to reproduce
+      description: >
+        What should we do to reproduce the problem? If you are not able to provide a reproducible case,
+        please open a [discussion](https://github.com/epam/ai-dial-chat/discussions) instead.
+      placeholder: >
+        Please make sure you provide a reproducible step-by-step case of how to reproduce the problem as minimally and precisely as possible.
+        Remember that non-reproducible issues will be closed! Opening a discussion is recommended as a first step.
+    validations:
+      required: true
```

Comment:
Not sure that this field should be required

Reply:
for bug report it's required